appveyor.yml

```diff
@@ -1,5 +1,9 @@
 environment:
 
+  TOXENV: py
```

Comment:
Use TOXENV=py which let tox use the Python interpreter used to invoke tox itself. Below we use `"%PYTHON%\\scripts\\tox -- --cov diffeqpy"` to invoke tox based on Python interpreter specified by `PYTHON` environment variable.